lib/tasks/correct_illness_from_csv.rake

```diff
@@ -0,0 +1,61 @@
+namespace :correct_illness do
+  desc "Export data CSVs for import into Central Data System (CDS)"
+  task :create_illness_csv, %i[organisation_id] => :environment do |_task, args|
+    organisation_id = args[:organisation_id]
+    raise "Usage: rake correct_illness:create_illness_csv['organisation_id']" if organisation_id.blank?
+
+    organisation = Organisation.find_by(id: organisation_id)
+    if organisation.present?
+      CreateIllnessCsvJob.perform_later(organisation)
+      Rails.logger.info("Creating illness CSV for #{organisation.name}")
+    else
+      Rails.logger.error("Organisation with ID #{organisation_id} not found")
+    end
+  end
+
+  desc "Export data CSVs for import into Central Data System (CDS)"
+  task :correct_illness_from_csv, %i[file_name] => :environment do |_task, args|
+    file_name = args[:file_name]
+
+    raise "Usage: rake correct_illness:correct_illness_from_csv['csv_file_name']" if file_name.blank?
+
+    s3_service = Storage::S3Service.new(PlatformHelper.is_paas? ? Configuration::PaasConfigurationService.new : Configuration::EnvConfigurationService.new, ENV["IMPORT_PAAS_INSTANCE"])
+    illness_csv = CSV.parse(s3_service.get_file_io(file_name), headers: false)
+
+    illness_csv.each_with_index do |row, index|
+      next if index < 3
```

Comment:
is this avoiding headers or something?

Reply:
Yup! There are 3 rows and I don't think there's a pretty way to skip them without using a gem